### PR TITLE
Reader: fix a few lint warnings

### PR DESCRIPTION
**Pull request by TooTallNate**
They were bothering me :smile_cat:

### CODE REVIEW

**Comment by mtias** [comment]
:) :+1:

File: client/reader/following-edit/style.scss [hunk 1]
```diff
@@ -146,7 +146,7 @@
 
 	.card.is-search-result {
 		position: absolute;
-			top: 51px;
+		top: 51px;
```
Comment (blowery) [inline client/reader/following-edit/style.scss]: These are on purpose :) 
We indent positioning rules for a reason I never understood. But we do.

Comment (nylen) [inline client/reader/following-edit/style.scss]: [Reference](https://github.com/Automattic/wp-calypso/blob/bdf36c9d45793157282b402f277bda6d471d2694/docs/coding-guidelines/css.md#positioning).  The idea is something like "these attributes are nested under the position" which makes conceptual sense to me.

Comment (blowery) [inline client/reader/following-edit/style.scss]: Yes, but every piece of tooling I throw at it wants to unindent them. :)

**Comment by blowery** [comment]
This was covered in other PRs that have already merged. :)

**Pull request closed without merging**